docs/UserGuide.md

```diff
@@ -274,20 +273,21 @@ Refer to [properties of contact](#properties-of-contact) on the accepted formats
 
 </box>
 
-###  Edit contact information: `edit`
+###  Edit information of contact: `edit`
 
 If the details of your contacts have changed, CampusConnect enables you to edit your contacts with the latest updated information so that you can still continue to keep in touch with them! Moreover, if you have made a mistake earlier while adding your contact information, CampusConnect empowers you to correct your mistakes through the `edit` command!
 
 This feature involves the command: `edit`, which edits existing information saved to your contact.
 
-Format: `edit PERSON_INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG] [tg/TELEGRAM] [e2/SECONDARY_EMAIL] [li/LINKEDIN] [b/BIRTHDAY]`
+Format: `edit PERSON_INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]... [tg/TELEGRAM] [e2/SECONDARY_EMAIL] [li/LINKEDIN] [b/BIRTHDAY]`
 
 <box type="info" icon=":fa-solid-magnifying-glass:">
 
 Below are some examples on how to use the command:
 
 * `edit 1 tg/@johndoe e2/johndoe@gmail.com`: Edits telegram to "@johndoe" and secondary email to "johndoe@gmail.com" for contact at index 1 of your contact list.   
 * `edit 2 n/Betsy e/betsy@example.com a/Newgate Heavan p/98765411 t/bestfriend`: Edits name to "Besty", email to "besty@example.com", address to "Newgate Heavan", phone to "98765411" and tag to "bestfriend" for contact at index 2 of your contact list.  
+* `edit 3 t/` Removes all existing tags for contact at index 3 of your contact list. <br> **Note: In this case, for the parameter `[t/TAG]`, you can only input once if you will like to remove all existing tags of your contact, e.g. `edit 3 t/ t/` will not work.**
```

Comment:
```suggestion
* `edit 3 t/` Removes all existing tags for contact at index 3 of your contact list. <br> **Note: In this case, for the `[t/TAG]` parameter, you can only input it once if you want to remove all existing tags from your contact, e.g. `edit 3 t/ t/` will not work.**
```
Consider rephrasing this like so.

Reply:
this phrasing sounds better! fixed in 255e904b